### ISSUE TITLE
Fill triggers not working

### ISSUE THREAD

**Issue by Bovrinox** [Stephan-S/FS22_AutoDrive#299]
It appears that AD is messing with the detection of fill triggers.
This happens for trailers and implements. The option to refill trailers from silos doesn't appear, same for filling seeders etc from pallets or big bags. 
![image](https://user-images.githubusercontent.com/63093402/151708542-19b7c55f-5625-4d5e-b1c5-03d887528e34.png)
This problem is solved buy exiting the vehicle, and getting back in while the tractor is in place. 
![image](https://user-images.githubusercontent.com/63093402/151708614-47affe6c-4d57-4243-b72b-50c72d474e67.png)
![image](https://user-images.githubusercontent.com/63093402/151708661-14089764-4fb1-425c-9322-2b3406bfa936.png)
As you can see, without moving the tractor the fill option has now appeared. It works the same way for seeders, fertilisers, slurry tanks etc. Anything that needs filled, basically.
Just for clarity, this is not while the tractor is being driven by AD. It is using the tractor manually. No other mods are installed, I updated to the latest release of AD before retesting and this bug is still present. I use a PS4 controller, but the issue is still present when using keyboard.

**Comment by Iwan1803** [comment]
I don't see any AD lines, waypoints or target points on the screenshots. There is also no target selected at the top of the HUD. Do you even have a route network there?

**Comment by Bovrinox** [comment]
no. Thats my point. Im not using autodrive, im trying to do it myself, but with autodrive active it messes with it.
"Just for clarity, this is not while the tractor is being driven by AD. It is using the tractor manually."

**Comment by Zwoxlol** [comment]
> no. Thats my point. Im not using autodrive, im trying to do it myself, but with autodrive active it messes with it. "Just for clarity, this is not while the tractor is being driven by AD. It is using the tractor manually."

Check if  you have double assignments on the Keyboard

**Comment by Bovrinox** [comment]
Not that I can see, no. 
Other than the fact that certain controls are contextual, and are the same button but used in different situations. Like press left stick fills the trailer, and also pets the doggo, depending on the situation.

**Comment by cater901645** [comment]
Ich glaub da is die Inputbinding.xml Schuld nicht AD, hatte das Prob selber schon mal, Inputbinding neu gemacht.

**Comment by Axel32019** [comment]
@Bovrinox Please make a backup of your inputBinding.xml, delete it and start the game new. Check...

**Comment by Bovrinox** [comment]
hi there, 
that seems to have done the trick, thanks so much! I shall bear it in mind for future, if i have any more bother with it.
I did notice while going through the options that it has controls for the controller for using AD, but only mapped for an xbox controller, so i didnt realise it had that option. I have now copied some of those controls to the corresponding buttons on the ps4 pad, and it shows them correctly in the top right button prompt window and everything!
Is it possible that the controls could be made default for both pads? Just a suggestion for the future.
Thanks again
Bov
